File: src/app/track.ts
export default class Track {

    public trackId: Number = 0;
    public trackName: string = '';
    public albumId: Number = 0;
    public albumName: string = '';
    public genreId: Number = 0;
    public genre: string = '';
    public milliSeconds: number = 0;

    constructor(trackId: Number, trackName: string, albumId: Number, albumName: string, genreId: Number, genre: string, milliseconds: number) {
        this.trackId = trackId;
        this.trackName = trackName;
        this.albumId = albumId,
            this.albumName = albumName,
            this.genreId = genreId,
            this.genre = genre,
            this.milliSeconds = milliseconds;
    }

}

File: server/PlaylistEditorV2/Models/Track.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace PlaylistEditorV2.Models
{
    public class Track
    {
        public int TrackId { get; set; }
        public string TrackName { get; set; }
        public int AlbumId { get; set; }
        public int GenreId { get; set; }
        public int Milliseconds { get; set; }



    }
}
